### ISSUE TITLE
How to pass extra options to the listen method

### ISSUE THREAD

**Issue by vsrnth** [rileyjhardy/deepgram#1]
Deepgram Listen API allows us to pass extra options for redaction, smart format etc.
- https://api.deepgram.com/v1/listen?smart_format=true&redact=ssn&diarize=true&language=en&model=nova-2

Is there way we can pass these extra options?

Thanks for creating this gem.

**Comment by rileyjhardy** [comment]
Yes, you should be able to pass in these options as keyword arguments to either of the `transcribe_file` or `transcribe_url` methods. They'll be converted into query parameters

```
def transcribe_file(path:, audio_format: 'audio/wav', **kwargs)
        validate_file_path(path)

        request(:post, **kwargs) do |request|
          request.headers['Content-Type'] = audio_format
          request.body = File.binread(path)
        end
      end
```
